src/kernel/emulation/linux/misc/sysctl_machdep.c

```diff
@@ -57,6 +57,15 @@ const struct known_sysctl sysctls_machdep[] = {
             unsigned int ecx
 #endif
 
+
+inline void copyout_int(int value, char* to_copy, int to_copy_leng)
```

Comment:
`static`

Reply:
`leng` is an unusual way to spell "length", it should be either `len` or `length`.

And it also shouldn't be an `int`, it should be `size_t` or `unsigned long`.